types/clarinet/index.d.ts

```diff
@@ -0,0 +1,98 @@
+// Type definitions for clarinet 0.12
+// Project: https://github.com/dscape/clarinet
+// Definitions by: corno <https://github.com/corno>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// import { Stream } from "stream";
```

Comment:
Did you intend to remove?

Reply:
I do not know how to import from the @types/node. Can you tell me how to?

Reply:
`///<reference types="node">`

Read the entry on [this page](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html#-reference-types-) to verify whether that is what you're looking for.

Reply:
The imports I commented on are commented out though, so it looks like they were left in on accident?

Reply:
well, they should have been active, but I did not know how to, so thank you for the pointer.

If I add ````///<reference types="node">```` It still reports that it cannot find module "stream"

Reply:
Sorry :(
```ts
/// <reference types="node"/>
import { Stream } from "stream";
```
I left out the final `/`

Reply:
Thank you. That did it. I'll create a pull request right now.